src/scripts/Build-Workbook.ps1

```diff
@@ -24,10 +24,47 @@ $Debug = $DebugPreference -eq "Continue"
 $outDir = Join-Path .. .. release "$Workbook-workbook"
 $workbookDir = Join-Path .. workbooks $Workbook
 
+# Worbkooks' templates placeholders
+$placeholders = @(
+    [PSCustomObject]@{
+        optimization = @{
+            "Compute"    = "computeGroupPlaceholder"
+            "Storage"    = "storageGroupPlaceholder"
+            "Networking" = "networkingGroupPlaceholder"
+            "AHUB"       = "AHUBGroupPlaceholder"
+        }
+    }
+)
+
 if (-not (Test-Path $workbookDir)) {
     return
 }
 
+# Check if the workbook is using templates
+if (Test-Path -Path $workbookDir -PathType Container) {
```

Comment:
Move this after the scaffolding updates and work off the file in the release folder. That'll be a cleaner place to manipulate code. That'll also allow us to track only a single `workbook.json` file.

Reply:
The current workflow is that with each build we generate a new workbook.json file based on the sub-templates in the workbook folder and then copy the generated workbook.json to the releases folder. So I think we need to do this before scaffolding as there won't be any workbook.json file initially.

Reply:
I understand that. My point is that doing it in the release folder is cleaner. Think about how binary compilers work: They generate things in an obj folder and output the results to the bin folder. We're effectively doing the same, but just using a single release folder for that. We shouldn't be changing anything in the src folder as part of the build.